apps/src/applab/dontMarshalApi.js

```diff
@@ -42,10 +83,17 @@ exports.removeItem = function (array, index) {
   if (index < array.length) {
     delete array.properties[array.length - 1];
     array.length -= 1;
+  } else {
+    // index is out of bounds (too large):
+    outputWarning("removeItem() index parameter value (" + index +
+                    ") is larger than the number of items in the list (" +
```

Comment:
I thought about that, but decided not to change the function's behavior. That might require a doc change and potentially break apps that were already written (thought I'd expect not).